groups/alteryx-live/iam.tf

```diff
@@ -30,7 +30,19 @@ module "alteryx_server_profile" {
       actions = [
         "cloudwatch:PutMetricData"
       ]
-    }
+    },
+    {
+      sid     = "S3AllowListGet"
```

Comment:
Can we split these into two separate statements so as to be able to keep the targets tightly defined